### PR TITLE
backend-setup

### PR DESCRIPTION
**Pull request by medha-prasad**

### CODE REVIEW

File: backend/server.js
Comment (sanjibbehera) [inline backend/server.js:1]: port no ko 8888 karlo plz from 3000

File: backend/src/config/config.js
Comment (sanjibbehera) [inline backend/src/config/config.js:1]: this file should read values from environment, something like this:-
account: process.env.SNOWFLAKE_ACCOUNT,
    username: process.env.SNOWFLAKE_USERNAME,
    database: process.env.SNOWFLAKE_DATABASE,
    schema: process.env.SNOWFLAKE_SCHEMA,
    warehouse: process.env.SNOWFLAKE_WAREHOUSE,
    password: process.env.SNOWFLAKE_PASSWORD,

**Review by sanjibbehera — COMMENTED** [review]
just adapt the port no and env variables.
rest is okay

**Comment by medha-prasad** [comment]
Updated the PR with the changes. Thanks

**Review by sanjibbehera — APPROVED** [review]

**Pull request merged**